stdlib/public/core/Span/Span.swift

```diff
@@ -437,6 +444,15 @@ extension Span where Element: ~Copyable {
 
 @available(SwiftStdlib 6.1, *)
 extension Span where Element: BitwiseCopyable {
+  @_semantics("fixed_storage.check_index")
+  @inline(__always)
+  @_alwaysEmitIntoClient
+  internal func _checkIndex(_ position: Index) {
+    _precondition(
+      UInt(bitPattern: position) <  UInt(bitPattern: _count),
+      "Index out of bounds"
+    )
+  }
 
```

Comment:
I think this one had the `UInt`-based comparison because I missed it when converting to `indices.contains()`. I don't see a reason to have a duplicate `_checkIndex()` for the `BitwiseCopyable` case.